plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoQueryRunner.java

```diff
@@ -57,7 +59,8 @@ public static DistributedQueryRunner createMongoQueryRunner(MongoServer server,
             Map<String, String> properties = ImmutableMap.of(
                     "mongodb.case-insensitive-name-matching", "true",
                     "mongodb.seeds", server.getAddress().toString(),
-                    "mongodb.socket-keep-alive", "true");
+                    "mongodb.socket-keep-alive", "true",
+                    "mongodb.credentials", "default");
```

Comment:
Do we need to pass actual credentials here?
```suggestion
                    "mongodb.credentials", MongoServer.getDefaultCredentials());
```